### ISSUE TITLE
Add a new organization to ROR: Kanagawa Cancer Center

### ISSUE THREAD

**Issue by adambuttrick** [ror-community/ror-updates#1808]
Summary of request: Add a new organization to ROR

Name of organization: Kanagawa Cancer Center
Website: https://kcch.kanagawa-pho.jp
Link to publications: 
Organization type: Healthcare
Wikipedia page: https://en.wikipedia.org/wiki/Kanagawa_Cancer_Center
Wikidata ID: Q11588998
ISNI ID: 0000 0004 0629 2905
GRID ID:
Crossref Funder ID:
Other names for the organization: 
Aliases:
Labels: 神奈川県立がんセンター*Japanese
Acronym/abbreviation:
Related organizations: 
City: Yokohama
Country: Japan
Geonames ID: 1848354
Year established: 1986
How will a ROR ID for this organization be used? 
Other information about this request:

**Comment by adambuttrick** [comment]
Cancer treatment facility in Japan. Funds and employs researchers. Solid affiliation usage. Should be added.

Publication affiliation usage: https://doi.org/10.1016/s0140-6736(17)31827-5; https://doi.org/10.1016/s1470-2045(18)30937-9; https://doi.org/10.1056/nejmoa1810865; https://doi.org/10.1056/nejmoa1709937; https://doi.org/10.1056/nejmoa1813904; https://doi.org/10.1056/nejmoa1809697; https://doi.org/10.1056/nejmoa1914510; https://doi.org/10.1245/s10434-011-2049-9; https://doi.org/10.1001/jama.2013.3190; https://doi.org/10.1056/nejmoa003034

ORCID affiliation usage: https://orcid.org/0000-0002-3718-9068 ; https://orcid.org/0000-0002-0804-7271 ; https://orcid.org/0000-0002-3940-9634

**Comment by mariagould** [comment]
I agree this is in scope given evidence of research activities and affiliation usage.
